tests/FakeItEasy.Analyzer.Tests.Shared/Helpers/DiagnosticVerifier.Helper.cs

```diff
@@ -1,3 +1,5 @@
+using System.Runtime.ExceptionServices;

```

Comment:
inside the namespace

Reply:
oops

Reply:
Fixed (I made a fixup and squashed immediately)